source/content/guides/composer-convert.md

```diff
@@ -6,49 +6,104 @@ permalink: docs/guides/:basename
 cms: "Drupal 8"
 categories: [develop]
 tags: [composer, site, workflow]
-contributors: [dustinleblanc]
+contributors: [dustinleblanc, stovak]
 ---
 
 ## Before You Begin
 
-- Review our documentation on [Git](/git), [Composer](/composer), and [Terminus](/terminus), and have them installed and configured on your local computer.
-- [Clone](/git#clone-your-site-codebase) your _current_ Pantheon site repository to a working directory on your local computer.
-- Review [Serving Sites from the Web Subdirectory](/nested-docroot)
-
 <Alert title="Exports" type="export">
 
 This guide uses the local command line environment, and there are several commands dependent on your specific site. Before we begin, set the variable `$site` in your terminal session to match your site name:
 
 ```bash{promptUser:user}
-export site=yoursitename
+export SITE=my-example-site
 ```
 
 </Alert>
 
+This document is for you if you meet the following criterion:
+
+- You have [Git](/git), [Composer](/composer), and [Terminus](/terminus), installed and configured on your local computer.
+
+  If not, you'll need to install these utilities for your specific operating system.
+
+- [You have a local copy of your site cloned from it's git repo](/git#clone-your-site-codebase) your _current_ Pantheon site repository in a working directory on your local computer.
+
+- Your site repository has a "/web" folder at it's root. 
+
+  [Serving Sites from the Web Subdirectory](/nested-docroot)
+
+- Your site has our DROPS-8 repo in it's upstream.
```

Comment:
If your site is based on drops-8, then it won't have a `web` folder at it's root.